core/tests/karma.conf.ts

```diff
@@ -148,25 +142,37 @@ module.exports = function(config) {
           'node_modules',
         ],
       },
+      devtool: 'inline-source-map',
       module: {
-        rules: [{
-          test: /\.ts$/,
-          use: [
-            'cache-loader',
-            'thread-loader',
-            {
-              loader: 'ts-loader',
-              options: {
-                // this is needed for thread-loader to work correctly
-                happyPackMode: true
+        rules: [
+          {
+            test: /\.ts$/,
+            use: [
+              'cache-loader',
+              'thread-loader',
+              {
+                loader: 'ts-loader',
+                options: {
+                  // this is needed for thread-loader to work correctly
+                  happyPackMode: true
+                }
               }
+            ]
+          },
```

Comment:
Do we still need this rule? Isn't it overwritten by the istanbul one?

Reply:
We still need it since the istanbul one only collects coverage.